rkt/image_list.go

```diff
@@ -32,12 +32,15 @@ import (
 const (
 	defaultTimeLayout = "2006-01-02 15:04:05.999 -0700 MST"
 
-	id         = "id"
-	name       = "name"
-	importTime = "import time"
-	lastUsed   = "last used"
-	size       = "size"
-	latest     = "latest"
+	id               = "id"
+	name             = "name"
+	importTime       = "import time"
+	lastUsed         = "last used"
+	size             = "size"
+	latest           = "latest"
+	verificationHash = "verification hash"
+	sourceURL        = "source URL"
+	insecureOptions  = "Insecure Options"
```

Comment:
Building this PR I'm not able to use these new fields and cannot see them in the help output. Probably because they aren't registered in the below `fields`. Perhaps some functional tests will be good.